openquake/hazardlib/shakemap/parsers.py

```diff
@@ -386,12 +390,32 @@ def download_station_data_file(usgs_id):
             stations_json_str = urlopen(stationlist_url).read()
             try:
                 stations = read_usgs_stations_json(stations_json_str)
-            except LookupError as exc:
-                logging.info(str(exc))
+            except (LookupError, UnicodeDecodeError,
+                    json.decoder.JSONDecodeError) as exc:
+                # TODO: return this also to the webui
+                logging.warning(str(exc))
+                return None
             else:
+                original_len = len(stations)
+                seismic_len = len(
+                    stations[stations['station_type'] == 'seismic'])
                 df = usgs_to_ecd_format(stations, exclude_imts=('SA(3.0)',))
                 if len(df) < 1:
-                    logging.warning('No seismic stations found')
+                    if original_len > 1:
+                        if seismic_len > 1:
+                            # TODO: return this info also to the webui
+                            logging.warning(
+                                f'{original_len} stations were found, but the'
+                                f' {seismic_len} seismic stations were all'
+                                f' discarded')
+                        else:
+                            # TODO: return this info also to the webui
+                            logging.warning(
+                                f'{original_len} stations were found, but none'
+                                f' of them are seismic')
+                    else:
+                        # TODO: return this info also to the webui
+                        logging.warning('No stations were found')
```

Comment:
This should be an `info` not a `warning` since it is a quite common case.

Reply:
Ok. I am doing this change in the next PR that I am opening now